spec/agent_spec.rb

```diff
@@ -1,16 +1,17 @@
+# frozen_string_literal: true
 require 'spec_helper'
 require 'ronin/web/browser/agent'
 
 describe Ronin::Web::Browser::Agent do
-  describe "#initialize" do
-    context "when Ronin::Support::Network::HTTP.proxy is set" do
+  describe '#initialize' do
+    context 'when Ronin::Support::Network::HTTP.proxy is set' do
```

Comment:
For some reason I always use double quoted strings in RSpec tests. Old habits die hard I guess.